test/test_rest_vol.c

```diff
@@ -1471,8 +1471,6 @@ test_unused_file_API_calls(void)
         hid_t               obj_id;
         void               *file_handle;
 
-        if (H5Fflush(file_id, H5F_SCOPE_GLOBAL) >= 0)
```

Comment:
I tested it with the flush tests in `vol-tests` to make sure it functions correctly.